glitchdotcom/Glitch-Community
# QA fixes

## Links
* https://cookie-ricotta.glitch.me
* https://app.clubhouse.io/glitch/story/8171/bug-fixes-from-qa

## GIF/Screenshots:

## Changes:
* Fixes helpdoc links so they open in new tabs
* Fixes a helpdoc link that pointed to the wrong spot
* Fixes Account Setting button logic so that it shows when Account Deletion toggle is enabled

## How To Test:
* Turn on Account Deletion (only)
* Make sure you can get there via user-options-pop
* Open the modal and make sure the "Learn about exporting" button takes you to https://glitch.com/help/how-can-i-download-backup-my-glitch-project/ in a new tab
* On the project transfer page, make sure top links open in new tabs
* On the team transfer page, make sure top links open in new tabs
